rts/motoko-rts/src/types.rs

```diff
@@ -213,7 +213,7 @@ impl Value {
     /// `Value::get_raw` and `unskew`) in our cost model where every Wasm instruction costs 1
     /// cycle.
     pub fn get(&self) -> PtrOrScalar {
-        if self.0 & 0b1 == 0b1 {
+        if self.0 & 0b1 != 0 {
```

Comment:
The `rustc` seems to generate identical code, but this form rings to me more like "the bit is set".